plugins/repository-s3/src/main/java/org/elasticsearch/plugin/repository/s3/S3RepositoryPlugin.java

```diff
@@ -144,6 +145,7 @@ public void onModule(SettingsModule settingsModule) {
         settingsModule.registerSetting(S3Repository.Repository.STORAGE_CLASS_SETTING);
         settingsModule.registerSetting(S3Repository.Repository.CANNED_ACL_SETTING);
         settingsModule.registerSetting(S3Repository.Repository.BASE_PATH_SETTING);
+        settingsModule.registerSetting(S3Repository.Repository.THROTTLE_RETRIES_SETTING);
```

Comment:
Something like `settingsModule.registerSetting(S3Repository.Repository.KEY_SETTING);` registers a global setting named  `access_key`: once done, nothing else (core or another plugin) can register a setting with the same name. And it laos registers it as `Filtered` so I guess every setting with the same name is filtered out from every places?

That seems wrong to me because the plugin only need to retrieve the `access_key` from repository settings and to filter out this setting from the repo settings. I think we should find another solution but not register this as a global setting like it is in the code.